### PR TITLE
Trokhymets Serhii Home Work 1

### PR DESCRIPTION
**Pull request by SerhiiTrokhymets**

### CODE REVIEW

**Comment by slenderock** [comment]
There are some critical fixes to make:
1. revert deleting other user profiles.
2. Use only English for comments in code as well as a folder name

Also, start using English for commit names. It is not critical for this PR but would be critical for next ones.

File: ihor.bohdan/first/task 2/hash.rb [hunk 1]
```diff
@@ -1,13 +0,0 @@
-# frozen_string_literal: true
```
Comment (slenderock) [inline ihor.bohdan/first/task 2/hash.rb]: please revert this change

File: ihor.bohdan/first/task 1/array.rb [hunk 2]
```diff
@@ -1,4 +0,0 @@
-# frozen_string_literal: true
```
Comment (slenderock) [inline ihor.bohdan/first/task 1/array.rb]: please revert this change

**Review by slenderock — CHANGES_REQUESTED** [review]

File: Трохимец Сергий/First HomeWork/Task2.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Task 2
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234',
+ yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+ p hash.count{ |key, _| key =~ /^yes/}
```
Comment (slenderock) [inline Трохимец Сергий/First HomeWork/Task2.rb]:
```suggestion
 p hash.count{ |key, _| key =~ /^yes/ }
```

https://github.com/rubocop-hq/ruby-style-guide#spaces-operators
It's better to add spaces so every part of your code is stand out visually

File: Трохимец Сергий/First HomeWork/Task1.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Task 1
+array = Array.new(3){ Array.new(3){rand(-10..10)}}.flatten # рандомний двовимірний масив (3 на 3)
```
Comment (slenderock) [inline Трохимец Сергий/First HomeWork/Task1.rb]: https://github.com/rubocop-hq/ruby-style-guide#spaces-operators
It's better to add spaces so every part of your code is stand out visually

```suggestion
array = Array.new(3){ Array.new(3){ rand(-10..10) } }.flatten # generating two dimensional array (3 x 3)
```

File: Трохимец Сергий/First HomeWork/Task1.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+p array
+array.each do |var|
+    if var > 0
+        puts var
+    end
+end
+
+array.reject!{ |x| x < 0} # залишаю в масиві тільки більше 0
```
Comment (slenderock) [inline Трохимец Сергий/First HomeWork/Task1.rb]:
```suggestion
array.reject!{ |x| x < 0 } # leave only positive numbers
```

File: Трохимец Сергий/First HomeWork/Task1.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+p array
+suma = array.inject(0){|result, element| result + element } # додаю значення масиву
```
Comment (slenderock) [inline Трохимец Сергий/First HomeWork/Task1.rb]:
```suggestion
suma = array.inject(0){ |result, element| result + element } # sum the array values
```

**Comment by slenderock** [comment]
As there are no major issues here, I'll give you some time to apply fixes, if you want. This request will be merged in 2 days

**Review by slenderock — APPROVED** [review]

**Comment by SerhiiTrokhymets** [comment]
> As there are no major issues here, I'll give you some time to apply fixes, if you want. This request will be merged in 2 days

For the future, all comment will be accepted
